README.md

```diff
@@ -8,12 +8,15 @@ application:ensure_all_started(hnc).
 {ok, _}=hnc:start_pool(my_pool, #{}, my_worker, []).
 
 %% Check out a worker.
-Worker=hnc:checkout(my_pool).
+WorkerRef=hnc:checkout(my_pool).
+
+%% Get the worker pid from the reference.
```

Comment:
Better call it "identifier" instead of "reference", as in the documentation.

Reply:
Done, I added a code line underneath for better illustration, too.

Reply:
Yes, that is better :)